internal/driver/service_test.go

```diff
@@ -0,0 +1,105 @@
+//
+// Copyright (C) 2022 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package driver
+
+import (
+	"fmt"
+	"sync/atomic"
+
+	dsModels "github.com/edgexfoundry/device-sdk-go/v2/pkg/models"
+	"github.com/edgexfoundry/device-sdk-go/v2/pkg/service"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/models"
+	"github.com/pkg/errors"
+)
+
+type MockSDKService struct {
+	devices map[string]models.Device
+	added   uint32
+	Config  map[string]string
+}
+
+func NewMockSdkService() *MockSDKService {
```

Comment:
Preference is to use mockery (https://github.com/vektra/mockery) to generate mocks from interface definitions.
Example: https://github.com/edgexfoundry/app-functions-sdk-go/blob/main/pkg/interfaces/mocks/ApplicationService.go

Reply:
This is legacy code from the LLRP repo that never got migrated to use mockery. I don't think we are actually using this mock yet, so maybe I will remove it for now.

Reply:
removed